pkg/bgs

```diff
@@ -0,0 +1,21 @@
+[mirrors]
+https://github.com/Gottox/bgs/archive/master.tar.gz
```

Comment:
please never ever use master- if you have to use bleeding edge (usually a release tag is preferable), then use the commit hash of master at that point in time. see pkg/john for an example.
we want our packages to have reproducible results.